Packages/com.unity.inputsystem/InputSystem/Devices/ScreenKeyboard.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Runtime.InteropServices;
+using UnityEngine.InputSystem.Utilities;
+
+
+namespace UnityEngine.InputSystem
+{
+    public enum ScreenKeyboardType
+    {
+        Default = 0,
+        ASCIICapable = 1,
+        NumbersAndPunctuation = 2,
+        URL = 3,
+        NumberPad = 4,
+        PhonePad = 5,
+        NamePhonePad = 6,
+        EmailAddress = 7,
+        Social = 8,
+        Search = 9
+    }
+
+    public enum ScreenKeyboardStatus
+    {
+        Visible,
+        Done,
+        Canceled,
+        LostFocus
+    }
+
+
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ScreenKeyboardShowParams
```

Comment:
Would prefer this as a protected struct hidden inside ScreenKeyboard.

Reply:
This is actually a struct used by user